common/element_handle.go

```diff
@@ -75,7 +75,7 @@ func (h *ElementHandle) checkHitTargetAt(apiCtx context.Context, point Position)
 			return false, err
 		}
 		if box == nil {
-			return false, errors.New("getting bounding box of element")
+			return false, errors.New("missing bounding box of element")
```

Comment:
Shouldn't this be:

```suggestion
			return false, errors.New("getting bounding box of element")
```

I thought errors describe the action that was occurring when something went wrong?

Reply:
Most often, that's right. However, this last one is not an action, it's the final result of that action. Similar to the ones that we output errors from CDP/injectedScript actions. It's the final piece in the error chain.

We get the following:

```
tapping element: checking hit target: missing bounding box of element
```

However, the following doesn't make much sense:

```
tapping element: checking hit target: getting bounding box of element
```

It's because we didn't try to get the bounding box, instead, we couldn't find the bounding box. The action is the "checking hit target".